cmd/monitor/streaming/bootstrap.yaml

```diff
@@ -371,6 +371,12 @@ storage-retention-strategy@metric:
   load_from_database: true
   ttl_reload_interval: "3m"
 
+storage-retention-strategy@external_metric:
+  # default 180 days
+  default_ttl: "${EXTERNAL_METRIC_TTL:4320h}"
```

Comment:
How to keep dop metrics always persisted?

Reply:
removed this  useless provider